### PR TITLE
add disabled to controls in which the video is already playing or paused

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR disables `play` button in the control menu when the video is already playing as well as disables the pause button in control menu when the Video is already paused
#### Where should the reviewer start?
video.js
#### What testing has been done on this PR?
storybook 
#### How should this be manually tested?
add `prop` to Video in storybook 

```
controls={{ position: "below", items: ["pause", "play"] }}

```
#### Do Jest tests follow these best practices?

- [ ] `screen` is used for querying.
- [ ] The correct query is used. (Refer to [this list of queries](https://testing-library.com/docs/queries/about/#priority))
- [ ] `userEvent` is used in place of `fireEvent`.
- [ ] `asFragment()` is used for snapshot testing.

#### Any background context you want to provide?
User plays or pauses the video via the extended "controls" prop. They are able to pause or play with any of the two controls. For example, the video will be playing and the user can go to the extended controls and hit the play icon. This will pause the video.
#### What are the relevant issues?
#6021 
#### Do the grommet docs need to be updated?
no
#### Should this PR be mentioned in the release notes?
maybe
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

**Comment by britt6612** [comment]
Need to look at TS issue

**Review by jcfilben — COMMENTED** [review]
I think we need to add disabled to the typescript definition for items. Right now it is just has icon, a11yTitle, and onClick
```
items: PropTypes.arrayOf(
  PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.shape({
      icon: PropTypes.element,
      a11yTitle: PropTypes.string,
      onClick: PropTypes.func,
    }),
  ]),
),
```

File: src/js/components/Video/stories/typescript/TheaterMode.tsx [hunk 1]
```diff
@@ -25,9 +25,9 @@ const TheaterMode = (props) => {
               a11yTitle: 'Toggle theater mode',
             },
             'fullScreen',
+            'pause',
```
Comment (jcfilben) [inline src/js/components/Video/stories/typescript/TheaterMode.tsx:28]:
```suggestion
```
I don't think we need to include the pause button in this example. Or if we do include it we should also have the play button.

Comment (britt6612) [reply src/js/components/Video/stories/typescript/TheaterMode.tsx:28]: aw yes no this was me testing TS

**Review by jcfilben — APPROVED** [review]
Looks good

File: src/js/components/Video/Video.js [hunk 2]
```diff
@@ -313,6 +313,7 @@ const Video = forwardRef(
             id: 'video.pauseButton',
             messages,
           }),
+          disabled: !playing,
```
Comment (ericsoderberghp) [inline src/js/components/Video/Video.js:316]: This seems to conflict with the code in the next line that checks for `playing` and calls `play` if not.

**Pull request merged**